project.clj

```diff
@@ -12,6 +12,7 @@
                  [org.cyverse/clojure-commons "3.0.3"]
                  [org.cyverse/heuristomancer "2.8.6"]
                  [org.flatland/ordered "1.5.7"]]
-  :eastwood {:linters [:wrong-arity :wrong-ns-form :wrong-pre-post :wrong-tag :misplaced-docstrings]}
+  :eastwood {:exclude-namespaces [common-swagger-api.schema.data.exists]
```

Comment:
First time I'm noticing this.  Why do we want to exclude namespaces from lint checking?

Reply:
In this case it's because we have the key `:/path/from/request/to/a/file/or/folder` in the `ExistenceResponsePathsMap` schema.

The `eastwood` linter complains that this is not a valid map key, but it compiles and displays in the docs just fine anyway 😕

Reply:
Ah, I see now - thanks for the explanation!